src/managed/Microsoft.Extensions.DependencyModel/CollectionExtensions.cs

```diff
@@ -36,5 +36,22 @@ private static IEnumerable<string> GetAssets(IEnumerable<RuntimeAssetGroup> grou
                 .Where(a => string.Equals(a.Runtime, runtime, StringComparison.Ordinal))
                 .SelectMany(a => a.AssetPaths);
         }
+
+        public static IEnumerable<RuntimeFile> GetDefaultRuntimeFiles(this IEnumerable<RuntimeAssetGroup> self) => GetRuntimeFiles(self, string.Empty);
```

Comment:
Review note: I'm not happy with this naming. These are mirrors of existing methods like `GetDefaultRuntimeAssets` but return RuntimeFile class instead of a string